### PR TITLE
Fix installation instructions in readme and docs

### PR DESCRIPTION
**Pull request by schustmi**
## Describe changes
During the TFX removal we forgot to update a few installation instructions.

## Pre-requisites
Please ensure you have done the following:
- [x] I have read the **CONTRIBUTING.md** document.
- [x] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/component-gallery/integrations) table and the [corresponding website section](https://zenml.io/integrations).
- [ ] I have added tests to cover my changes.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other (add details above)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
 [![PyPi][downloads-shield]][downloads-url]
 [![Contributors][contributors-shield]][contributors-url]
 [![License][license-shield]][license-url]
-[![Build][build-shield]][build-url]
+<!-- [![Build][build-shield]][build-url] -->
```
Comment (fa9r) [inline README.md:15]: what does this do?

**Review by fa9r — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (schustmi) [reply README.md:15]: It removes the `Build: Failing` badge on our Readme. This currently points to some CI workflow on develop which is just too flaky, not sure why that should be displayed on the `main` branch Readme which is always passing as it's pointing to a release.

**Review by htahir1 — APPROVED** [review]

**Comment by htahir1** [comment]
@schustmi Please backport this when you merge this

**Pull request merged**